front/app/containers/Admin/reporting/containers/ReportBuilderPage/index.tsx

```diff
@@ -23,11 +22,10 @@ import messages from './messages';
 import { isNilOrError } from 'utils/helperUtils';
 
 const ReportBuilderPage = () => {
-  const reportBuilderEnabled = useFeatureFlag({ name: 'report_builder' });
   const reports = useReports();
   const [modalOpen, setModalOpen] = useState(false);
 
-  if (!reportBuilderEnabled || isNilOrError(reports)) {
+  if (isNilOrError(reports)) {
     return null;
```

Comment:
The button that's show when showEmptyState is false should also have this.

Reply:
Indeed, odd state to be in though 🤔. It would mean a report was already created but now they can't create one anymore. Happy to add it

Reply:
> Indeed, odd state to be in though 🤔. It would mean a report was already created but now they can't create one anymore. Happy to add it

Yes, this covers the path where the customer downgrades from premium to standard. Not common, but happened before I believe.